posthog-core/src/index.ts

```diff
@@ -1823,14 +1833,14 @@ export abstract class PostHogCore extends PostHogCoreStateless {
       this.capture('$feature_flag_called', {
         $feature_flag: key,
         $feature_flag_response: response,
-        $feature_flag_id: featureFlag?.metadata?.id,
-        $feature_flag_version: featureFlag?.metadata?.version,
-        $feature_flag_reason: featureFlag?.reason?.description ?? featureFlag?.reason?.code,
-        $feature_flag_bootstrapped_response: bootstrappedResponse,
-        $feature_flag_bootstrapped_payload: bootstrappedPayload,
+        $feature_flag_id: featureFlag?.metadata?.id ?? null,
```

Comment:
This changes behavior, right? Previously we'd not include the key when sending to the server, we now include the key with `null` value.

Reply:
mm good point, not sure if we're dropping `key: undefined` or fetch would drop it, will test and fix if needed